DMCompiler/Compiler/DM/DMLexer.cs

```diff
@@ -5,8 +5,9 @@
 using OpenDreamShared.Compiler;
 
 namespace DMCompiler.Compiler.DM {
-    public partial class DMLexer : TokenLexer {
-        public static List<string> ValidEscapeSequences = new() {
+    public sealed class DMLexer : TokenLexer {
+        // NOTE: .NET still needs you to pass the capacity size to generate the most optimal code, so update it when you change these values
+        public static readonly List<string> ValidEscapeSequences = new(41) {
```

Comment:
Is it really much of a concern when the variable is static? It's only every going to be created once, when the application starts.

Reply:
Is there any reason _not_ to? How often does this list really get edited and how much work is it to just change the number?

Reply:
Fair enough